<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>本地表情库</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="name_frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="localPath_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>D:\studiolibrary</string>
        </property>
        <property name="cursorPosition">
         <number>16</number>
        </property>
        <property name="placeholderText">
         <string extracomment="0">本地表情库路径</string>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::VisualMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="localPath_label">
        <property name="text">
         <string>本地表情库路径</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="refresh_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="lib_local_root_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>本地根目录</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="readonly_local_checkBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>只读</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="scene_scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>163</width>
           <height>266</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="scene_characters_label">
            <property name="text">
             <string>本场景角色</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="scene_characters_textScrollList"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="local_scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>162</width>
           <height>266</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="local_characters_label">
            <property name="text">
             <string>本地角色</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="local_characters_textScrollList"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
